### ISSUE TITLE
[a11y] Generate label on save is missing

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1244]
**Describe the bug**
Generate label on save does not appear on the form.

**To Reproduce**
1. Go to the CO form (`/specify/view/collectionobject/5327/` for freshfish)
2. See missing item

**Expected behavior**
![image](https://user-images.githubusercontent.com/37256050/163589673-a1808660-2b45-4487-b996-70f3da3456ba.png)
Using 'edge'

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/163589771-62787ce2-084f-407e-af64-d6d08dba20e6.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Edge

**Comment by maxxxxxdlp** [comment]
The cell is hidden in the definition (visible=false):
```xml
<cell colspan="2" default="true" id="113" ignore="true" initialize="visible=false" label="Generate Label on Save" name="generateLabelChk" type="field" uitype="checkbox" />                 
```

Current release did not support that attribute, which is why the field is visible on edge. See https://github.com/specify/specify7/issues/1070
